banzai/main.py

```diff
@@ -259,17 +277,19 @@ def reduce_night():
 
     telescopes = dbs.get_telescopes_at_site(pipeline_context.site,
                                             db_address=pipeline_context.db_address,
-                                            must_be_schedulable=pipeline_context.telescopes_must_be_schedulable)
+                                            must_be_schedulable=not pipeline_context.ignore_schedulability)
```

Comment:
Oops, this was my bad! But maybe it would be less confusing if `must_be_schedulable` is changed to `ignore_schedulability` as well, i.e. to avoid passing a `not`.

Reply:
Ok. Done.